docs/docs/configuration/auth.md

```diff
@@ -12,6 +12,7 @@ Valid providers are :
 - [ADFS](#adfs-auth-provider)
 - [Facebook](#facebook-auth-provider)
 - [GitHub](#github-auth-provider)
+- [Gitea](#gitea-auth-provider)
```

Comment:
Should this not be before GitHub alphabetically?

Reply:
As it isn't a full provider in itself, I decided to put it below GitHub. I can move it up if you want me to?